### DISCUSSION TITLE
Not generate world "TERMUR" help? thanks

### DISCUSSION THREAD

**Discussion by Lito2022** [modernuo/ModernUO#1697] [General]
I'm trying to create a place to mess around and see if I can set up a place, but I can't activate it so that the entire world is generated, including underworld, termur, champs PERLess, can anyone help me?

**Comment by kamronbatman** [comment]
When you are in-game, use `[admin` then go to World Building to add stuff. Not all content (especially newer expansions) are added to ModernUO. Would love help in adding the missing content!
